packages/vite/src/devFeServer.ts

```diff
@@ -80,6 +80,30 @@ async function createServer() {
       : route.pathDefinition
 
     app.get(expressPathDef, createServerAdapter(routeHandler))
+
+    app.all(
+      '/_rw_mw',
```

Comment:
I wanted to ask actually - what if we intercepted all POST requests (apart from `/rw-rsc` or defined function endpoints? We won't need this url.

Reply:
> heir own custom/special endpoints to their Fastify config?

No I don't think it would. Because any functions/endpoints will take precedence over middleware. 

> Are there any cons you can think of if doing that?

Just not sure about RSC/server actions related stuff - not sure what endpoint they'd be hitting. I suppose middleware would need to run for them anyway

Reply:
I'm down to try making it a catch-all kind of thing if you want. As long as it's just in canary we can always make it more specific if we find out we need to